pkg/api/admin/openshiftcluster_validatestatic.go

```diff
@@ -0,0 +1,298 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"net"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/immutable"
+	"github.com/Azure/ARO-RP/pkg/util/subnet"
+)
+
+var (
+	rxResourceGroupID = regexp.MustCompile(`(?i)^/subscriptions/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/resourceGroups/[-a-z0-9_().]{0,89}[-a-z0-9_()]$`)
```

Comment:
These look like material to be placed in `pgk/api` and re-used as they should not change. Or change across the board.

Reply:
@mjudeikis Please take a look at https://github.com/Azure/ARO-RP/pull/145. It changes location of the validators and these variables as well.

Reply:
I'll update this PR once we settle on appraoch for #145